### PR TITLE
implement trim function

### PR DESCRIPTION
**Pull request by ptvnbvn1**

### CODE REVIEW

File: src/string/trim.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+export const trim = function (str, re) {
+  var i;
+  var demo;
+
+  if (re == null) {
+    str = str.replace(/ /g, '');
+    return str;
+  }
+  str = str.replace(/ /g, '');
+  for (i = 0; i <= re.length; i++) {
```
Comment (shirai91) [inline src/string/trim.js]: if you are going to use i inside this for only, please use es6 's `let` keyword
example:
```javascript
for(let i=0; i<=re.length;i++) {
// do your works
}
```

Comment (ptvnbvn1) [reply src/string/trim.js]: thanks, I fixed it.

File: src/string/trim.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+export const trim = function (str, re) {
+  var demo;
+
+  if (re == null) {
+    str = str.replace(/ /g, '');
```
Comment (shirai91) [inline src/string/trim.js]: if you are going to use white space character (` `) for space, please consider to use `\s` character to matching white space. as all modern regex use `\s` to matching white space, and we should not use ` ` character in regex pattern

Comment (ptvnbvn1) [reply src/string/trim.js]: Thanks you, I fixed it =))

**Pull request merged**